server/index.js

```diff
@@ -18,15 +18,60 @@ app.use('/:menu', express.static(path.join(__dirname, '../client/dist')));
 app.use(parser.json());
 app.use(parser.urlencoded({ extended: false }));
 
+
+app.get('/api/menu', function (req, res) {
+     MenuItem.getAllMenus()
+     .then(data => res.status(202).send(data))
+     .catch(err => console.log(`API Error:${err}`));
+ });
+
+app.post('/api/menu/', function (req, res) {
+    MenuItem.createMenu(req.body)
+     .then(data => res.status(202).send(data))
+     .catch(err => console.log(`API Error:${err}`));
+   });
+
 app.get('/api/menu/:menu', function (req, res) {
   const menu = parseInt(req.params.menu);
   if (menu > 100) {
```

Comment:
I know you didn't write this, but at some point you're going to need to fix it. It might be worth fixing right now, if it wouldn't take you too long.